core/src/main/scala/com/snowplowanalytics/snowplow/eventgen/protocol/Body.scala

```diff
@@ -76,32 +76,32 @@ object Body {
     );
   private def genWithEt(
     etGen: Gen[EventTransaction],
-    now: Instant,
+    time: Instant,
```

Comment:
Does your PR change the logical meaning of this Instant?  Or did you change the name from `now` to `time` just because you didn't like the original name?

Reply:
Just the name changes. It was called `now` by legacy but it is possible to pass the timestamp that we want in the config.